server/src/main/java/org/opensearch/action/ActionListenerResponseHandler.java

```diff
@@ -89,4 +90,10 @@ public Response read(StreamInput in) throws IOException {
     public String toString() {
         return super.toString() + "/" + listener;
     }
+
+    @Override
+    public Response read(InputStream in) throws IOException {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'read'");
```

Comment:
Can you provide a default implementation in the interface that throws this unsupported operation exception to avoid having to place these stubs all over the code base?

Reply:
Done